examples/django_example/README.md

```diff
@@ -0,0 +1,15 @@
+# Django Amplitude example
+
+Contains a app called 'amplitude_example'. The app has one view in amplitude_example/views.py
```

Comment:
I think it would be better to make each subproject README standalone. Can you please add the Amplitude SDK install and API key notes here as well as in the top level README.